docs/getting-started/creating-plugins.md

```diff
@@ -35,6 +41,53 @@ The `json` file contains the following structure:
 }
 ```
 
+#### Definition structure
+
+The `Definition` property has the following structure:
+
+```json
+{
+  "Id": "Id shall contain a stable identifier for the rule",
+  "Name": "Name may contain an identifier that is understandable to an end user",
+  "HelpUri": "HelUri may contain the absolute URI of the primary documentation for the reporting item",
+  "Message": "Message may contain the default message that will be used",
+  "Kind": "ResultKind of the issue",
+  "Level": "FailureLevel of the issue",
+  "Description": "Description of the rule",
+  "FileNameDenyRegex": "Regular expression that will be denied during analysis",
```

Comment:
>analysis [](http://example.com/codeflow?start=70&length=8)

This sentence and the one that follows isn't clear.

'Any files whose names match the 'FileNameDenyRegex' pattern will be added to a deny list (i.e., they won't be scanned).'

'Any files whose names match the 'FileNameAllowRegex' will be added to an allow list (i.e., they will be scanned)'